src/main/java/com/hms/flexyosisoftconnector/OSIsoftConnectorMain.java

```diff
@@ -30,10 +30,10 @@
 public class OSIsoftConnectorMain {
 
   /** Application Major Version Number */
-  static final int MAJOR_VERSION = 2;
+  static final int MAJOR_VERSION = 3;
 
   /** Application Minor Version Number */
-  static final int MINOR_VERSION = 9;
+  static final int MINOR_VERSION = 0;

```

Comment:
What is using these version numbers? If it is simply to log out the version of the connector, I suggest you implement pulling that from Maven.

Reply:
yes it was, added